### ISSUE TITLE
Switch keyboard focus color incorrect.

### ISSUE THREAD

**Issue by JacobDFrank** [adobe/spectrum-css#445]
## Description

The keyboard focus color for the switch in each theme is off. In each theme, the color of the before and after elements within the spans is blue 500 when it should be blue 400.

## Steps to reproduce

1. Go to http://spectrum-css-archives.s3-website.us-east-2.amazonaws.com/2.15.1/docs/toggle.html
2. Keyboard focus by tabbing over the switches
3. The colors are off.

## Expected behavior

The colors for keyboard focus should be blue 400

## Environment
 - **Spectrum CSS version:** 
 - **Browser(s) and OS(s):**

**Comment by lazd** [comment]
Cannot reproduce, I have `blue-400` for everything when focused:

![image](https://user-images.githubusercontent.com/201344/70738004-ae357f80-1cc8-11ea-8cda-2bcb7f4ec9d9.png)

**Comment by JacobDFrank** [comment]
Could not be reproduced. Closing...
